src/Raven.Client/Data/PutResult.cs

```diff
@@ -3,13 +3,12 @@
 //     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
 // </copyright>
 //-----------------------------------------------------------------------
-using System;
 
-namespace Raven.Abstractions.Data
+namespace Raven.Client.Data
```

Comment:
Don't change namespaces of existing classes without talking to someone about it, this will break clients code